### PR TITLE
btrfs-progs: new package at 5.2

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

**Comment by shizonic** [comment]
@konimex Better like this?

**Comment by konimex** [comment]
You don't really have to close the 3 PRs. I was just asking you to put the PRs at the same time as there was significant delay between `lzo` and your main `btrfs-progs` PR.

Also, a commit generally should only contain changes for one package.

**Comment by shizonic** [comment]
Ahh I missunderstood. Sorry about. So should I split it again into single PRs or may I let it with this one?

**Comment by konimex** [comment]
This PR is fine, but split the commits; one for `zstd`, one for `lzo` and one for `btrfs-progs`. Just force push afterwards.

**Comment by shizonic** [comment]
Done.

**Comment by konimex** [comment]
Please fix the shellcheck errors.

File: public/btrfs-progs/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+NOCONFIGURE=1 ./autogen.sh
+
+# Requires losetup which is part of util-linux, but even with it installed it fails
```
Comment (dylanaraps) [inline public/btrfs-progs/build]: Split this into two shorter lines and add a full stop.

File: public/btrfs-progs/build [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+rm -rf -- tests/mkfs-tests/017-small-backing-size-thin-provision-device
```
Comment (dylanaraps) [inline public/btrfs-progs/build]: I don’t think — is needed here.

File: public/lzo/build [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```
Comment (dylanaraps) [inline public/lzo/build]: Split this into two lines.

File: public/btrfs-progs/build (hunk 2 above)
Comment (shizonic) [reply public/btrfs-progs/build]: So if I remove it the above comment is obsolete, isn't it?

Comment (dylanaraps) [reply public/btrfs-progs/build]: Just remove the “—“. It’s not needed.

Comment (shizonic) [reply public/btrfs-progs/build]: Ahh I see. Will do all changes asap. I'm currently out walking with my dog.

**Comment by shizonic** [comment]
Changes are done

**Comment by dylanaraps** [comment]
LGTM :+1:

**Pull request merged**